SoftLayer/CLI/file/modify.py

```diff
@@ -0,0 +1,70 @@
+"""Modify an existing file storage volume."""
+# :license: MIT, see LICENSE for more details.
+
+import click
+import SoftLayer
+from SoftLayer.CLI import environment
+from SoftLayer.CLI import exceptions
+
+
+CONTEXT_SETTINGS = {'token_normalize_func': lambda x: x.upper()}
+
+
+@click.command(context_settings=CONTEXT_SETTINGS)
+@click.argument('volume-id')
+@click.option('--new-size', '-c',
+              type=int,
+              help='New Size of file volume in GB. '
+                   '***If no size is specified, the original size of '
+                   'volume will be used.***\n'
+                   'Potential Sizes: [20, 40, 80, 100, 250, '
+                   '500, 1000, 2000, 4000, 8000, 12000] '
+                   'Minimum: [the size of the origin volume] '
+                   'Maximum: [the minimum of 12000 GB or '
+                   '10*(origin volume size)]')
+@click.option('--new-iops', '-i',
+              type=int,
+              help='Performance Storage IOPS, between 100 and 6000 in '
+                   'multiples of 100 [only used for performance volumes] '
+                   '***If no IOPS value is specified, the original IOPS value of the '
+                   'volume will be used.***\n'
+                   'Requirements: [If original IOPS/GB for the volume is less '
+                   'than 0.3, new IOPS/GB must also be less '
+                   'than 0.3. If original IOPS/GB for the volume is greater '
+                   'than or equal to 0.3, new IOPS/GB for the volume must '
+                   'also be greater than or equal to 0.3.]')
+@click.option('--new-tier', '-t',
+              help='Endurance Storage Tier (IOPS per GB) [only used for '
+                   'endurance volumes] ***If no tier is specified, the original tier '
+                   'of the volume will be used.***\n'
+                   'Requirements: [IOPS/GB for the volume is 0.25, '
```

Comment:
This is minor, but for consistency, I think it would be helpful to insert "If original" before "IOPS/GB" here.